packages/compass-e2e-tests/tests/oidc.test.ts

```diff
@@ -278,6 +309,11 @@ describe('OIDC integration', function () {
   });
 
   it('can successfully re-authenticate', async function () {
+    // TODO: not working for multiple connections
```

Comment:
This doesn't pop up a modal or anything for multiple connections, but it does for single connection.